versioned_docs/version-8.7/self-managed/operational-guides/backup-restore/zeebe-backup-and-restore.md

```diff
@@ -175,6 +175,25 @@ curl --request GET 'http://localhost:9600/actuator/backups'
 
 </details>
 
+### Request to list backups matching a prefix
+
+The list of backups can be filtered by specifying a backup ID prefix:
+
+```
+GET actuator/backupRuntime/{backupIdPrefix}
+```
+
+The backup ID prefix must end with `*`, for example `10*` will match all backups with IDs starting with `10`.
```

Comment:
❓ 💭  Should we add, what happens when it is not ended with `*`? Like it will interpreted as normal ID, and likely return nothing.

Reply:
I think the existing docs + the error response of "No backup with id $id" and 404 response is clear enough.